notify/twitter_notify.py

```diff
@@ -14,18 +17,18 @@
 https://gist.github.com/blacktwin/261c416dbed08291e6d12f6987d9bafa
 """
 
-from twitter import *
+from twitter import Twitter, OAuth
```

Comment:
It _appears_ these are the only objects being used from here, but I haven't verified this still works.